cranelift-codegen/meta/src/isa/x86/opcodes.rs

```diff
@@ -263,6 +263,9 @@ pub static PADDUSB: [u8; 3] = [0x66, 0x0f, 0xdc];
 /// Add packed unsigned word integers from xmm2/m128 and xmm1 saturate the results (SSE).
 pub static PADDUSW: [u8; 3] = [0x66, 0x0f, 0xdd];
 
+/// Bitwise AND of xmm2/m128 and xmm1 (SSE2).
```

Comment:
Why do you mention specific registers instead of just the register class (xmm)?

Reply:
It's not specific registers; I think in the Intel manual they choose to use this (sort of confusing) nomenclature for indicating "first XMM operand" and "second XMM operand." Following what I think @sstangl intended for this file I've been copying the text directly from the Intel manual.